README.md

```diff
@@ -371,7 +371,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 [OneWire]: https://crates.io/crates/onewire
 [SSD1322]: https://crates.io/crates/ssd1322
 [GridEYE]: https://github.com/uwearzt/grideye
-[AT24Cx]: https://github.com/eldruin/at24cx-rs
+[EEPROM24x]: https://github.com/eldruin/eeprom24x-rs
 [DS1307]: https://github.com/eldruin/ds1307-rs
```

Comment:
Since you're already touching DS1307, would you mind changing the link from GH to crates.io?

Reply:
Sure, I'll do that.